widgets/MapWidget/README

```diff
@@ -0,0 +1,16 @@
+MapWidget README Last Change: 07/13/2015 by greenertux (Tristan Hoffmann)
+
+### Intro ###
+This widget can be used to visualize GPS coordinates. For any list of documents of a certain entity that contain coordinates, you can get a shiny map.
+See sample.jpg for a preview.
+
+### How to use ###
+1) Create a custom widget in an entity list view in AppBuilder and paste the *.erb code into it
+2) Download jquery-jvectormap-1.2.2.min.js and jquery-jvectormap-world-mill-en.js from http://jvectormap.com/download/ and place them in your webserver directory (usually /path/to-your-installation/AppBuilder/wlp/usr/servers/AppBuilder/apps/AppBuilder/assets)
+3) Replace your-entity with the name of the entity you want to visualize
+4) Replace LocationLat and LocationLon with the field names containing latitude and longitude information
```

Comment:
Line numbers and sample code might be useful.  You may also want to think about changing your ERB code so that it's easier to reference exactly what needs to be changed.  For example, pulling out values to be changed into variables or putting parameters on different lines instead of all on one line maybe.